### DISCUSSION TITLE
Repostiory "Metadata"

### DISCUSSION THREAD

**Discussion by tripleo1** [community/community#13204] [Other Feature Feedback, Questions, & Ideas]
In the "About" section on the right side, I see way too many projects with `No description, website, or topics provided.`

I suggest to encourage people to add these things like you suggest adding a README (and do you suggest adding a LICENSE?)

**Comment by npanzica-qontigo** [comment]
I really like the idea of adding a description and website URL to the project but I cannot figure out how to add this information.  Maybe other people have that problem, too?  I am going to post a separate thread about this, but if anyone sees this thread and knows the answer, can you post something about how this is done? Some information I saw online said there should be some sort of cog icon to click but I do not see one. Are my permissions on the project too low to see this? I have edited and uploaded other documents there so I don't think so.

![image](https://user-images.githubusercontent.com/85634248/181655936-e071e23f-8065-4400-b40d-fb9a29127870.png)

**Reply by radar-blip** [reply]
There should be a little 'gear' icon next to the 'About' (to the right). Click on that to add info, description, etc.

![gear](https://user-images.githubusercontent.com/39196767/181656828-f8bb5ea8-af25-40a9-9006-3de80bab27cb.png)

**Reply by npanzica-qontigo** [reply]
Thank you, linuxlawson. Since you can see this, it gives me hope that I will eventually see it but right now...  there just doesn't seem to be a gear there. (It's not in the screen shot I attached, either).  I may try a machine reboot and then revisit the page. Maybe that will help. 

I think your About image holds a clue but I don't know how to interpret it. Your image shows that "About" is above some text that says "Frequently Used GitHub Markdown." The About section I am looking at doesn't have that text below the word "About" (see my screenshot above) so I might be on a different page than you are. Hmm...

**Reply by radar-blip** [reply]
That is the description I put there myself. You should  be able to add your own description also. Dont know why yours isnt showing up.

**Reply by radar-blip** [reply]
@npanzica-qontigo, I cant think of any reason you wouldn't have a gear icon there other than your repository being private, but I dont think that matters either. Maybe a glitch of some kind. If you want you could ask staff directly. Maybe they can put it back for you. 

Here is link to file a bug report: https://support.github.com/contact/bug-report

**Reply by tripleo1** [reply]
@npanzica-qontigo First thing first, are you logged in, and do you have permissions?

**Reply by jfullstackdev** [reply]
look at the one on top, it's not you, 

![image](https://user-images.githubusercontent.com/47092464/181685613-7caa175f-1c40-4247-b1d2-b65f4c5a88e4.png)

that user should have write access in order to have that Gear button

it does not match any of those contributors at that repo 

![image](https://user-images.githubusercontent.com/47092464/181685780-2e2d3ac7-e41f-4712-ad00-d491e6e2526b.png)

**Reply by radar-blip** [reply]
You're right, I thought I saw something odd but couldn't pin-point it. Seems someone is (or isn't) logged into the right account.

So, the original creator needs to get signed on in order to get gear button..?

**Reply by jfullstackdev** [reply]
yes, the user should have write access to that or else, anyone can edit that description, even us,

**Reply by radar-blip** [reply]
Well, good eye on catching that. Hopefully the owner of repo comes back and sees this.

**Comment by radar-blip** [comment]
@tripleo1, I read somewhere that you should always include a License. No matter the size or importance of the repo.

**Reply by tripleo1** [reply]
@linuxlawson I agree. I like a good `Makefile` too.

And a footrub now that I think of it...

**Comment by npanzica-qontigo** [comment]
Thank you all very much for the responses. Much appreciated!  This was very helpful, and I think the suggestion of their being an "owner" of this account is likely the reason why I can't access or see certain things, like a gear icon next to "About."  I'll investigate this further with those working on this project.

**Reply by radar-blip** [reply]
You're welcome. Hope you figure things out.
